### DISCUSSION TITLE
Tags instead of collections, or both maybe.

### DISCUSSION THREAD

**Discussion by Projosh00** [ppy/osu#30682] [Ideas] [answered]
I think it'd be easier to organize the beatmaps I want to play and practice regularly if we had tags or nested tags to be able to give to the maps.

Imagine this:
I might want to play practice maps for finger control, I can filter maps by the tags I set, in this case I'd search for:
_**Practice | Finger Control**_ 
_**Practice**_ being a tag and _**Finger Control**_ being a subtag or **tag that is nested** in the _**practice**_ tag.

And say I found a lot of maps, because of the time I would've spent tagging my maps, And I wanted some that I with a punchy beat, to help boil down the search, so I'd search something like:
_**Practice | Finger Control**, **Banger Maps**_

I think this would help me a lot when I want to look for something speific, but maybe I don't know the name, or I want something random from maps that can help me to what I want to do, not just something TRULY random.

This could also help when choosing maps for multiplayer, where you know what kinds of maps could bring some fun to the table, but you don't really have a collection for those maps.

**Comment by Joehuu** [comment, marked as answer]
There are efforts to bring custom user-defined online tags that do this, see https://github.com/ppy/osu-web/pull/11629 and https://github.com/ppy/osu-web/issues/8300

**Reply by Projosh00** [reply]
I assume this is only on the osu website, no?

Would this also be something users can add to maps or just something the mapper can add to ease the search of new maps?

**Reply by Joehuu** [reply]
Maybe it may eventually be integrated to song select, like what online features such as the favorites filter if you've ever used stable but is not yet implemented in lazer. Workarounds for online web features like those would be using the in-game beatmap listing / profile for now.

> Would this also be something users can add to maps or just something the mapper can add to ease the search of new maps?

As said in the linked pull request, users will be able to add a tag after they have a score on it.
